presentation/src/main/java/me/androidbox/presentation/event/viewmodel/EventViewModel.kt

```diff
@@ -159,12 +168,14 @@ class EventViewModel @Inject constructor(
             is EventScreenEvent.CheckVisitorExists -> {
                 verifyVisitorEmail(eventScreenEvent.visitorEmail)
             }
+            is EventScreenEvent.OnSaveEventID -> {
+                fetchEventById(eventScreenEvent.eventId)
+            }
```

Comment:
Why do you need this?
You named it "OnSaveEvent" meaning it should save, but you are fetching it

Reply:
@MKiperszmid This has been removed already. It was something I was trying when resolving this issue